lib/realtime_web/channels/realtime_channel.ex

```diff
@@ -501,6 +518,13 @@ defmodule RealtimeWeb.RealtimeChannel do
   defp is_new_api(%{"config" => _}), do: true
   defp is_new_api(_), do: false
 
+  defp is_public(params) do
+    case params["config"]["broadcast"]["public"] do
+      nil -> true
+      public -> !!public
+    end
```

Comment:
I think this would be equivalent to:

    params["config"]["broadcast"]["public"] != false